components/welcome/Welcome.module.css

```
@@ -0,0 +1,13 @@
+.background {
+  background: #50c4b7;
+  display: grid;
+  justify-items: center;
+  height: 100vh;
+  align-items: center;
+  grid-auto-rows: min-content;
+}
+.headline {
+  font-family: "AlmondCream", sans-serif;
+  color: #707070;
+  font-size: 60px;
```

You should set the size with em :-)

```suggestion
  font-size: 60px;
```